core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java

```diff
@@ -184,5 +186,7 @@ interface Key {
     String INVALID_OBJECT_TYPE = "0072";
     String INACTIVE_USER = "0073";
     String INVALID_CSV_FILE = "0074";
+    String EXTENDED_USER_PROFILE_NOT_LOADED = "EXTENDED_USER_PROFILE_NOT_LOADED";
```

Comment:
This should be error number as per new impl

Reply:
Updated the code with error code